0-bootstrap/versions.tf

```diff
@@ -15,5 +15,5 @@
  */
 
 terraform {
-  required_version = "~> 0.12.6"
+  required_version = ">=0.13, <0.14"
```

Comment:
Do you think it might be worthwhile continuing to support terraform > 0.12.6 until we start using terraform 0.13 features?